src/tools/std20/views/test/drop.test.cpp

```diff
@@ -0,0 +1,208 @@
+// include Catch2
+#include <catch2/catch_test_macros.hpp>
+
+// what we are testing
+#include "tools/std20/views.hpp"
+
+// other includes
+#include <forward_list>
+#include <list>
+#include <vector>
+#include "tools/std20/algorithm.hpp"
+
+// convenience typedefs
+//using namespace njoy::tools;
+namespace std20 = nano;
+
+SCENARIO( "drop_view" ) {
+
+  const std::vector< int > equal = { 0, 1, 2 };
+  const std::vector< int > modified = { 0, 2, 4 };
+  const std::vector< int > original_modified = { -2, -1, 0, 2, 4 };
+
+  GIVEN( "a container with forward iterators" ) {
+
+    std::forward_list< int > values = { -2, -1, 0, 1, 2 };
+
+    WHEN( "when iterators are used" ) {
+
+      auto chunk = values | std20::views::drop( 2 );
+      using Range = decltype(chunk);
+      using Iterator = nano::iterator_t< Range >;
+
+      THEN( "the take_view satisfies the required concepts" ) {
+
+		    CHECK( std20::ranges::view< Range > );
+		    CHECK( ! std20::ranges::sized_range < Range > );
+        CHECK( std20::ranges::forward_range< Range > );
+        CHECK( ! std20::ranges::bidirectional_range< Range > );
+		    CHECK( ! std20::ranges::random_access_range < Range > );
+		    CHECK( ! std20::ranges::contiguous_range < Range > );
+		    CHECK( std20::ranges::common_range < Range > );
+      }
+
+      THEN( "the take_view range and iterator associated types are correct" ) {
```

Comment:
"take_view" -> "drop" 

Likewise for other THENs in scenario

Reply:
fixed in fix/review